plugins/org.locationtech.udig.ui/src/org/locationtech/udig/ui/graphics/SWTGraphics.java

```diff
@@ -353,12 +353,14 @@ public void setTransform( AffineTransform transform ) {
         transform.getMatrix(matrix);
         if( swtTransform==null ){
             swtTransform=new Transform(display, 
-                    (float)matrix[0], (float)matrix[1], (float)matrix[2], 
-                    (float)matrix[3], (float)matrix[4], (float)matrix[5] );
+                    (float)matrix[0], (float)matrix[2],
+                    (float)matrix[1], (float)matrix[3],
+                    (float)matrix[4], (float)matrix[5] );
```

Comment:
Hi @akater320, 
I cannot understand why you have changed the ordering of  elements and put matrix[2] before matrix[1] in the code above. Can you clarify?

Reply:
Yeah, sorry, this commit message got lost in my fumble to squash and sign-off this branch.
java.awt.geom.AffineTransform.getMatrix(double[]) returns the 2x3 matrix in column-major order.
The org.eclipse.swt.graphics.Transform constructor expects the elements in the somewhat odd form of a 2x2 row-major matrix followed by a scale vector.

Reply:
OK thanks for the explanation